recipes/libgit2/0.28.x/conanfile.py

```diff
@@ -67,20 +67,20 @@ def _need_mbedtls(self):
         return "mbedtls" in (self.options.with_https, self.options.with_sha1)
 
     def requirements(self):
-        self.requires("zlib/1.2.12")
+        self.requires("zlib/1.2.13")
```

Comment:
```suggestion
        self.requires("zlib/[>=1.X.Y <2]")
```

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
@uilianries 
Thanks!
I fixed.